src/test/java/com/github/paweladamski/httpclientmock/HttpClientResponseBuilderTest.java

```diff
@@ -286,15 +286,12 @@ public void should_not_throw_exception_when_body_matcher_is_present_on_post_requ
     HttpResponse response = httpClientMock.execute(httpGet("http://localhost:8080/path2"));
     Assert.assertThat(response, hasStatus(SC_NOT_FOUND));
   }
-
+  
   @Test
-  public void doReturnFormParams_should_returnResponseEntityWithFormParameters() throws IOException {
+  public void doReturnFormParams() throws IOException {
```

Comment:
Please don't change this test name and body.

Reply:
I don't know what happened here. Maybe I should re-submit my pull request.

Reply:
Fixed in 459f3036b2aedfe6e1d3ee27e10c127729ce52ba. I restored your version of the file.

I think the file got messed up when I attempted to your branch into my branch after 1.7.0 release. Sorry about that!